content/english/useless-box/build-the-box.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Build the box"
+description: ""
+draft: true
+weight: 3
+---
+
+## Let�s build the box
+![diagram of the parts to build the box, base, lids, sides, end, motor mount and finger](img/boxPartsDiagram.png)
+1.	Glue 2 fingers together, to make a thicker finger. (If it�s already thick enough, skip this step)
+2.	Attach the motor to the motor mount
+3.	Attach the momentary/limit switch 
+4.	Mount the toggle switch onto the lid
+5.	Press the finger onto the motor shaft and secure it
+6.	The rest of the box slides and snaps together.
+	a.	Fit the motor mount piece to the base, try to get the finger to be centered in the box
+	b.	Fit the sides
+	c.	Put one lid on and snap the end. 
+Should look something like this:
+![4 images showing steps to build the box](img/boxBuildingSteps.png)
+
+Here�s the circuit design:
+![diagram of the final circuit](img/circuit.png)
+When the limit switch connects and something (for example: you) changes the position of the toggle switch ON, the battery powers the motor, which moves the finger.  
+The mechanical finger from inside the box comes out and toggles the switch OFF, so the mechanical finger retracts.
+
+
+### *CONGRATS! You�ve done it. Feel free to play with your box and learn or experiment with more basic circuits.*
+
+Supporting resources: 
+
+1. http://makezine.com/projects/the-most-useless-machine/
+2. http://www.youtube.com/watch?v=aqAUmgE3WyM
+
+
+---
+![Nuevo Foundation robot mascot](img/NF_mascot.jpg)
```

Comment:
This workshop completed section should be filled in automatically once the workshop is published